content/js2/blocks/events/index.md

```diff
@@ -22,11 +22,17 @@ However, we're missing a step in our plan. We need to find a way of running some
 An [event](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events) is something that occurs in a programming environment.
 {{</note>}}
 
-Events are things that happen in the browser, which the browser tells you about so your code can react to them. In a browser context, events could be a user clicking on a button, a user typing something into a textarea box, submitting a form etc. You can find a [complete reference all the different event types](https://developer.mozilla.org/en-US/docs/Web/Events) on MDN.
+Events are things that happen in the browser, which your code can ask to be told about, so that your code can react to them. In a browser context, an event could be a user clicking on a button, a user typing something into a textarea box, a user submitting a form etc. Not all events are in response to user actions, for instance there is an event for the browser finished initially rendering the page. You can find a [complete reference all the different event types](https://developer.mozilla.org/en-US/docs/Web/Events) on MDN.
 
-In the case of the `textarea` element, there are lots of events occurring. The `textarea` value is changing, a keyboard button is released every time the user types and so on. We can update our plan as follows:
```

Comment:
I don't think it was salient here to describe that a key-press triggers multiple events, it felt like a distraction. It's definitely interesting and useful, but probably not required here?